fix: coordinates bug on pdf parsing (#1462)
Unstructured-IO/unstructured: CHANGELOG.md

## 0.10.16

> 

Reviewer: ```suggestion
```
rm line

### Fixes

> * **GH Issue 1460** Fixes a bug that prevented certain pdf's from bypassing new coordinate logic

Reviewer: ```suggestion
* ***Fixes an issue that caused a partition error for some PDF's.** Fixes GH Issue 1460 by bypassing a coordinate check if an element has invalid coordinates.
```

### Fixes

> * ***Fixes an issue that caused a partition error for some PDF's.** Fixes GH Issue 1460 by bypassing a coordinate check if an element has invalid coordinates.

Reviewer: ```suggestion
* ***Fixes an issue that caused a partition error for some PDF's.** Fixes GH Issue 1460 by bypassing a coordinate check if an element has invalid coordinates.

```
newline
